hunting/index.md

```diff
@@ -6,3 +6,60 @@ Here are the queries currently available:
 - [Denial of Service or Resource Exhaustion Attacks Detection](./llm/docs/llm_dos_resource_exhaustion_detection.md) (ES|QL)
 - [Monitoring for Latency Anomalies](./llm/docs/llm_latency_anomalies_detection.md) (ES|QL)
 - [Sensitive Content Refusal Detection](./llm/docs/llm_sensitive_content_refusal_detection.md) (ES|QL)
+
+
+## macos
```

Comment:
maybe we have missed to pull and convert this one https://github.com/elastic/ia-trade-team/issues/302#issuecomment-1964665824

Reply:
https://github.com/elastic/detection-rules/pull/3642/files#diff-18c167bbd3c8dcf9e6818e685188c350369a1712c928dff95f70323c4033c262R8